dartFlutter/Dockerfile.debug

```diff
@@ -0,0 +1,51 @@
+# ARGUMENTS --------------------------------------------------------------------
+##
+# Base Container version
+# Using Flutter v3.22.0
+##
+ARG SDK_BASE_VERSION=early
+##
+# Base container version
+##
+ARG BASE_VERSION=runtime_debug
+
+##
+# Board architecture
+##
+ARG IMAGE_ARCH=
+
+##
+# Directory of the application inside container
+##s
+ARG APP_ROOT=
+
+# BUILD/DEPLOY -----------------------------------------------------------------------
+##
+# Build/Deploy Step
+##
+FROM crossware.app:5050/toradex/flutter:${SDK_BASE_VERSION} as build
```

Comment:
Hello @alberrizwan,
Sorry for taking a bit of time to the PR review, we are on the process of a new major release of Torizon (Torizon 7).
I created a project form the template and tried to build it, but fell into the following issue:
```
 *  Executing task: DOCKER_HOST= docker compose build --pull --build-arg SSHUSERNAME=torizon --build-arg APP_ROOT=/home/torizon/app --build-arg IMAGE_ARCH=arm64 --build-arg DEBUG_SSH_PORT= --build-arg GPU=-vivante testflutter-debug 

WARN[0000] The "DOCKER_LOGIN" variable is not set. Defaulting to a blank string. 
WARN[0000] /home/andreriesco/Documents/IDE2.0/test-applications/testFlutter/docker-compose.yml: the attribute `version` is obsolete, it will be ignored, please remove it to avoid potential confusion 
[+] Building 2.6s (3/3) FINISHED                                        docker-container:sharp_murdock
 => [testflutter-debug internal] load build definition from Dockerfile.debug                      0.3s
 => => transferring dockerfile: 1.23kB                                                            0.0s
 => WARN: FromAsCasing: 'as' and 'FROM' keywords' casing do not match (line 26)                   0.3s
 => WARN: FromAsCasing: 'as' and 'FROM' keywords' casing do not match (line 36)                   0.3s
 => ERROR [testflutter-debug internal] load metadata for crossware.app:5050/toradex/flutter:runt  2.1s
 => CANCELED [testflutter-debug internal] load metadata for crossware.app:5050/toradex/flutter:e  2.1s
------
 > [testflutter-debug internal] load metadata for crossware.app:5050/toradex/flutter:runtime_debug:
------
failed to solve: crossware.app:5050/toradex/flutter:runtime_debug: failed to resolve source metadata for crossware.app:5050/toradex/flutter:runtime_debug: failed to authorize: failed to fetch anonymous token: unexpected status from GET request to https://crossware.app/jwt/auth?scope=repository%3Atoradex%2Fflutter%3Apull&service=container_registry: 403 Forbidden

 *  The terminal process "/usr/bin/bash '-c', 'DOCKER_HOST= docker compose build --pull --build-arg SSHUSERNAME=torizon --build-arg APP_ROOT=/home/torizon/app --build-arg IMAGE_ARCH=arm64 --build-arg DEBUG_SSH_PORT= --build-arg GPU=-vivante testflutter-debug'" terminated with exit code: 17. 
 *  Terminal will be reused by tasks, press any key to close it. 
 ```
 It seems that the base container is not publicly available.

Reply:
Hello @alberrizwan,
Any updates on this issue?

Reply:
Hello @andreriesco,
Sorry for the delayed response, I have made the requested changes please check and let us know if any more changes are required.